components/Voting/EloVote.styled.tsx

```diff
@@ -5,21 +5,26 @@ import { calcRem } from 'components/utils/styles/calcRem'
 
 export const StyledEloVoteContainer = styled('div')(({ theme }) => ({
   display: 'grid',
-  gridTemplateColumns: `repeat(2, minmax(${calcRem(80)}, 1fr))`,
-  gridTemplateRows: '100%',
+  gridTemplateColumns: `repeat(1, minmax(${calcRem(80)}, 1fr))`,
   gridColumn: '1 / -1',
   gap: theme.metrics.md,
   inlineSize: '100%',
   maxInlineSize: calcRem(965),
-  maxBlockSize: '60vh',
+  /*maxBlockSize: '60vh',*/
   paddingInlineStart: calcRem(theme.metrics.sm),
   paddingInlineEnd: calcRem(theme.metrics.sm),
   marginInlineStart: 'auto',
   marginInlineEnd: 'auto',
+  overflowY: 'auto',
+  maxHeight: '75vh',
 
   [breakpoint('md')]: {
     gap: theme.metrics.lg,
   },
+  [breakpoint('sm')]: {
+    gridTemplateColumns: `repeat(2, minmax(${calcRem(80)}, 1fr))`,
+    maxHeight: 'none',
```

Comment:
Removing the max-height means that on desktop, some abstracts are long enough to overflow the page, which means we need to scroll down to the vote buttons. If that's the case on desktop/tablet, etc, why do we bother with the sticky buttons on mobile?

![need to scroll to buttons on desktop](https://user-images.githubusercontent.com/9972287/168457581-d51f4f2d-3e3a-44cb-8ccf-fa58a9acbe00.png)

Reply:
I found the sticky buttons on mobile a bit nicer since you can vote at any scroll position between the two talks (eg: if you've just scrolled back up to double check the first one, you can submit your vote).

I unbounded it on tablet and mobile since I find scrolling internally in the cards not a great experience, so if we have the screen real estate to display them side by side then it's nice to use it.

We can remove that rule if I'm in the minority here, but just wanted to touch on my rationale.